backend/src/resolvers/post.resolver.ts

```diff
@@ -0,0 +1,24 @@
+import { QueryResolvers, MutationResolvers } from '../generated/graphqlgen';
+
+export const PostQueryResolver: Pick<QueryResolvers.Type
+    , 'posts'
+    > = {
+    // @ts-ignore
+    posts: (parent, _args, { dataSources }) => {
+        return dataSources.databaseAPI.getAllPosts();
+    }
+};
+
+export const PostMutationResolver: Pick<MutationResolvers.Type
+    , 'write'
+    | 'upvote'
+    > = {
+    // @ts-ignore
+    write: (parent, { post }, { dataSources }) => {
+        return dataSources.databaseAPI.writePost(post);
+    },
+    // @ts-ignore
+    upvote: (parent, {id, voter}, { dataSources }) => {
+        return dataSources.databaseAPI.upvotePost(id, voter);
+    }
+};
```

Comment:
ahhhh these are your resolvers ... ok...
but we don't understand your tmp-resolver folder then ?! 

one resolver File would probably be enough ... a lot of import and export needed in you structure